requirements.txt

```diff
@@ -32,6 +32,7 @@ scikit-learn==0.24.1
 scispacy==0.4.0
 sentence-transformers==1.0.4
 spacy==3.0.6
+spacy-transformers==1.0.2
```

Comment:
Because we use `spaCy` models in the code under `src/bluesearch`.

To `load` a model with a transformer component, we need the transformer-related code from `spacy-transformers`.

Reply:
I see! I did not think about that! Thanks for the answer :)